dockerfiles/platform-build-incremental/Dockerfile

```diff
@@ -22,6 +22,12 @@
 ARG PREV_LABEL
 FROM hpccsystems/platform-build:${PREV_LABEL}
 
+RUN apt-get install -y \
```

Comment:
It's trying to reduce the number of times it is downloaded when running incr.sh - where it was it would still run every time you build for a new commit